electron-app/package.json

```
@@ -195,29 +199,29 @@
   },
   "theiaPluginsDir": "plugins",
   "theiaPlugins": {
-    "vscode-builtin-cpp": "https://open-vsx.org/api/vscode/cpp/1.52.1/file/vscode.cpp-1.52.1.vsix",
-    "vscode-arduino-api": "https://github.com/dankeboy36/vscode-arduino-api/releases/download/0.1.2/vscode-arduino-api-0.1.2.vsix",
+    "vscode-builtin-cpp": "https://open-vsx.org/api/vscode/cpp/1.88.1/file/vscode.cpp-1.88.1.vsix",
+    "vscode-arduino-api": "https://github.com/dankeboy36/vscode-arduino-api/releases/download/0.2.1/vscode-arduino-api-0.2.1.vsix",
```

```suggestion
    "vscode-arduino-api": "https://github.com/dankeboy36/vscode-arduino-api/releases/download/0.1.2/vscode-arduino-api-0.1.2.vsix",
```

Thank you! However, it depends on your release plan. I would be happy to see the new API integrated into the IDE nightly, and everything should be compatible, but I can't guarantee anything. If any issues arise, I'm available to help resolve them, or you can revert to a previous version if needed.

---

I propose reverting the changes for now, and after the IDE release, I will create a follow-up PR. Thank you!